### ISSUE TITLE
Some blocks have time stamps from the future

### ISSUE THREAD

**Issue by erikd** [input-output-hk/cardano-db-sync#276]
Just noticed this:
```
cexplorer=# select slot_no, epoch_no, epoch_slot_no, time from block
              where slot_no is not null order by slot_no desc limit 10 ;
 slot_no | epoch_no | epoch_slot_no |        time         
---------+----------+---------------+---------------------
 7535664 |      215 |         18864 | 2025-05-09 06:32:51
 7535653 |      215 |         18853 | 2025-05-09 06:29:11
 7535637 |      215 |         18837 | 2025-05-09 06:23:51
 7535610 |      215 |         18810 | 2025-05-09 06:14:51
 7535583 |      215 |         18783 | 2025-05-09 06:05:51
 7535556 |      215 |         18756 | 2025-05-09 05:56:51
 7535529 |      215 |         18729 | 2025-05-09 05:47:51
 7535502 |      215 |         18702 | 2025-05-09 05:38:51
 7535475 |      215 |         18675 | 2025-05-09 05:29:51
 7535448 |      215 |         18648 | 2025-05-09 05:20:51
(10 rows)
```
which is obviously way wrong. Odd thing is that only blocks in epoch 208 (the first Shelley epoch I think) had dates in the future.

Because rollbacks do not yet work in an time effective way, I added dropped the database and restarted the sync process with added debug.

**Comment by erikd** [comment]
@rhyslbw has confirmed that this is a regression against the last released version (`4.0.0`).

**Comment by erikd** [comment]
Not actually fixed until I get the dependencies updated.

**Comment by erikd** [comment]
Closed in #282 .
